Client/Frontend/Settings/LoginDetailViewController.swift

```diff
@@ -324,12 +324,14 @@ extension LoginDetailViewController {
         // Keep a copy of the old data in case we fail and need to revert back
         let oldPassword = login.password
         let oldUsername = login.username
-        login.update(password: password, username: username)
+        login.password = password
+        login.username = username
 
         if login.isValid.isSuccess {
-            profile.logins.updateLoginByGUID(login.guid, new: login, significant: true)
+            _ = profile.logins.update(login: login)
```

Comment:
+1 to `_ =` for  indicating this is an async op and it is intentional to let it run without needing to wait for a result.